geomesa-convert/geomesa-convert-metrics-ganglia/src/main/scala/org/locationtech/geomesa/convert/metrics/ganglia/GangliaReporterFactory.scala

```diff
@@ -48,10 +49,14 @@ class GangliaReporterFactory extends ReporterFactory {
 
 object GangliaReporterFactory {
 
-  case class GangliaConfig(group: String, port: Int, addressingMode: String, ttl: Int, ganglia311: Boolean)
+  import pureconfig.generic.semiauto._
 
   val GangliaDefaults: Config =
     ConfigFactory.empty
         .withValue("addressing-mode", ConfigValueFactory.fromAnyRef("unicast"))
         .withValue("ganglia311", ConfigValueFactory.fromAnyRef("true"))
+
+  implicit val GangliaConfigReader: ConfigReader[GangliaConfig] = deriveReader[GangliaConfig]
+
+  case class GangliaConfig(group: String, port: Int, addressingMode: String, ttl: Int, ganglia311: Boolean)
```

Comment:
Does this need to be done for the AWS CloudWatch config?

Reply:
good call, let me check, seems to have passed travis so maybe not

Reply:
it wasn't using pureconfig, but I updated it to do so as it gives nicer error messages.